cinn/ir/ir.h

```diff
@@ -474,10 +479,11 @@ struct LoadStoreAddrMnger {
  */
 struct Load : public ExprNode<Load>, public LoadStoreAddrMnger {
   std::vector<Expr> indices;
+  Expr index_;
```

Comment:
The `index_` should be a private member.

But IR node's member should be all public, or some state will differ after copy or visit.